pkg/source/file/multilines.go

```diff
@@ -141,7 +142,7 @@ func (mh *MultiHolder) flush() {
 	if mh.currentSize <= 0 {
 		return
 	}
-	mh.state.ContentBytes = mh.currentSize + 1
+	mh.state.ContentBytes = mh.currentSize + int64(len(globalLineEnd.GetEncodeLineEnd(mh.state.PipelineName, mh.state.SourceName)))
```

Comment:
`int64(len(globalLineEnd.GetEncodeLineEnd(mh.state.PipelineName, mh.state.SourceName)))` can be initialized in the `newMultiHolder`